ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -35,26 +35,46 @@ def toProtocolElement(self):
 
 
 class SimulatedPhenotypeAssociationSet(AbstractPhenotypeAssociationSet):
-    def __init__(self, parentContainer, localId, randomSeed):
+    def __init__(self, parentContainer, localId, randomSeed,
+                 numAssociations=2):
```

Comment:
`numAssociations` should be able to be set in `serverconfig.py`.  There should be a variable for that and also a variable for the number of `SimulatedPhenotypeAssociationSet`s in a dataset.  These variables should be passed through `frontend`, `backend`, down to the appropriate level.

We do this so we can easily tweak the simulator from the config -- without having to dive into the depths of the implementation -- for testing various conditions.